lib/paper_trail/has_paper_trail.rb

```diff
@@ -127,6 +127,18 @@ def paper_trail
         ::PaperTrail::RecordTrail.new(self)
       end
 
+      def dependent_versions(class_name, foreign_key)
+        children = []
+        versions = PaperTrail::Version.where(item_type: class_name)
```

Comment:
I feel like this code could be cleaner like so:

```ruby
def dependent_versions(class_name, foreign_key)
  PaperTrail::Version.where(item_type: class_name).
    subsequent(created_at).map(&:reify).select do |obj|
      obj.try(foreign_key) == id
    end
  end
end
```

Also it should probably have some logic to take polymorphous associations into account which is missing at the moment, correct?